src/main/resources/reference.conf

```diff
@@ -71,10 +71,20 @@ cassandra-journal {
   max-result-size = 50001
 
   # Dispatcher for the plugin actor.
-  plugin-dispatcher = "akka.actor.default-dispatcher"
+  plugin-dispatcher = "cassandra-journal.default-dispatcher"
```

Comment:
this is related to #137 , it's an intermediate solution until we have revisited all blocking calls (which is a bigger task)